src/app/pages/send/send-crypto-asset-form/components/amount-field.tsx

```diff
@@ -139,6 +152,7 @@ export function AmountField({
             fontWeight={500}
             autoComplete={autoComplete}
             {...field}
+            onChange={onChange}
```

Comment:
I'm fairly sure that adding the `onChange` here overwrites the value in `field`, breaking formik?

Isn't there a way to continue using the `useEffect` with the new logic?

Reply:
It is overriding it, but it doesn't break Formik as I'm passing the event to it in line 84 -> `field.onChange(event)`. Ideally, useEffect should be use as little as possible, here's an excellent resource from the new React docs https://react.dev/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state